docs/community/software/index.mdx

```diff
@@ -11,5 +11,7 @@ Current community projects:
 
 - [Meshtasticator (Simulator)](/docs/community/software/community-meshtasticator) - Meshtasticator is a discrete-event and interactive simulator that mimics the radio section of the device software.
 - [Meshtastic Web API](/docs/community/software/meshtastic-web-api) - Meshtastic Web API provides a RESTful interface to interact with a Meshtastic node via a serial connection.
+- [Mesh-metrics](/docs/community/software/community-mesh-metrics) - Meshtastic repeater metrics scraper using meshttastic-cli to be ingrested by node_exporter for use in Prometheus//VictoriaMetrics TDB with a Grafana dashboard and sample alerting rules.
 
 Support for these projects should be sought from their respective authors.
+community-mesh-metrics
```

Comment:
this looks like an error

Reply:
I removed that extra line 17. Likely a paste error.